queries.sql

```diff
@@ -8,3 +8,54 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*1*/
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+
+/*2*/
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals ORDER BY id ASC;
+
+/*3*/
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals ORDER BY id ASC;
+ROLLBACK;
+SELECT * FROM animals ORDER BY id ASC;
+
+/*4*/
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals ORDER BY id ASC;
+
+/*4*/
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, COUNT(*) FROM animals WHERE escape_attempts > 0 GROUP BY neutered;
```

Comment:
👍 Good effort in working on the query! To fully meet the project requirement of determining who escapes the most, whether neutered or not, please consider the following improvements:

- [ ] Update your query so that it accurately calculates and compares the number of escape attempts for both neutered and non-neutered animals. Ensure that the query results clearly show neutered or not, has the highest number of escape attempts. Currently, you are counting the animals based on neutered whose escape_attempts is greater than 0. Add ScreenShot in the PR too.

| Your | query |
| --- | --- |
|  ![image](https://github.com/lerfast/vet_clinic_database/assets/55840999/41bbcc47-201d-485a-bf9a-0cc4318e3ded)| ![image](https://github.com/lerfast/vet_clinic_database/assets/55840999/8db0cd10-b0af-4e3b-9916-beb11aef90bc)|


By making this adjustment, your query will provide the required information as per the project's specifications. Keep up the great work! 👍📊🐾

Reply:
![image](https://github.com/lerfast/vet_clinic_database/assets/111823606/f41a034a-a7a7-485e-9a98-84327be709d8)
done, thank you for notice that!!!